src/frame.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+class Frame {
+  constructor () {
+    this.bowlOne = 0;
+    this.bowlTwo = 0;
+    this.MAX_SCORE = 10;
+    this.isStrike = false;
+    this.isSpare = false;
+    this.score = 0;
+  }
+
+  addBowlOneScore(pins) {
+    if(pins === this.MAX_SCORE) {
+      this.isStrike = true;
+    }
+    this.bowlOne += pins;
+    this.calculateTotal();
+  };
+
+  addBowlTwoScore(pins) {
+    this.bowlTwo += pins;
+    this.calculateTotal();
+    if(this.calculateTotal() === 10) {
```

Comment:
Could we refactor this to avoid having [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming))? Are there other places in the code where we refactor magic numbers?